pandas/tools/util.py

```diff
@@ -61,6 +61,42 @@ def to_numeric(arg, errors='raise'):
         - If 'raise', then invalid parsing will raise an exception
         - If 'coerce', then invalid parsing will be set as NaN
         - If 'ignore', then invalid parsing will return the input
+    downcast : {'signed', 'unsigned'}, str, numpy.dtype, or
+               Python dtype, default None
```

Comment:
@gfyoung I don't think putting this on multiiple lines will work (limitations of sphinx unfortunately ...). Although I should test to be sure. 
It's a bit a problem with the unflexibility of sphinx and having to adhere to PEP8 to satisfy our linter ..

You can maybe just say "data type" instead of "str, numpy dtype or Python dtype" and then put in the explanation that the data type can be a string, numpy dtype or Python type.